src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsregisterstonomisupdate/services/CourtRegisterUpdateService.kt

```diff
@@ -275,7 +304,7 @@ class CourtRegisterUpdateService(
           postalCode = building.postcode,
           county = getRefCode("COUNTY", building.county),
           country = getRefCode("COUNTRY", building.country),
-          primary = courtDto.buildings.size == 1 || building.subCode != null,
+          primary = building == buildings[0], // first one in the list?
           noFixedAddress = false,
```

Comment:
would it be first with no subcode?

Reply:
No, ones with sub codes will be be in a different object